### PR TITLE
Add missing features for HTMLInputElement

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR is a part of a project to add missing interfaces and interface features to BCD that are from a spec in Editor's Draft or more and is supported in at least one browser.  This particular PR adds missing features, populating the results using data from the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project (v1.1.6), for the HTMLInputElement API.

Spec: https://html.spec.whatwg.org/multipage/semantics.html#htmlinputelement
IDL: https://github.com/w3c/webref/blob/master/ed/idl/html.idl

Notice: the diff looks pretty weird for this one due to the similarities in the results between all of the features.  This PR doesn't modifying any existing features, however.

### CODE REVIEW

**Comment by foolip** [comment]
This PR is huge, and will also require checking that reflected IDL attributes have their content attribute data updated. @vinyldarkscratch how about splitting this into smaller pieces, where the diff of each piece becomes more managable?

File: api/HTMLInputElement.json [hunk 1]
```diff
@@ -95,33 +94,32 @@
           }
         }
       },
-      "autofocus": {
+      "align": {
```
Comment (foolip) [inline api/HTMLInputElement.json:97]: `align` is listed in https://html.spec.whatwg.org/multipage/obsolete.html#HTMLInputElement-partial and should be deprecated. (But I can't make suggestions in that part of the diff.)

File: api/HTMLInputElement.json [hunk 2]
```diff
@@ -1155,60 +2625,59 @@
...
-              "version_added": true
+              "version_added": "1"
             }
           },
           "status": {
-            "deprecated": false,
             "experimental": false,
-            "standard_track": true
+            "standard_track": true,
+            "deprecated": false
```
Comment (foolip) [inline api/HTMLInputElement.json:2671]:
```suggestion
            "deprecated": true
```

https://html.spec.whatwg.org/multipage/obsolete.html#HTMLInputElement-partial

File: api/HTMLInputElement.json [hunk 3]
```diff
@@ -191,47 +188,44 @@
           }
         }
       },
-      "files": {
+      "autocapitalize": {
```
Comment (foolip) [inline api/HTMLInputElement.json:191]: `autocapitalize` shouldn't be added: https://github.com/mdn/browser-compat-data/issues/7519

**Review by foolip — COMMENTED** [review]
Took another look, but this is still very large and difficult to review. Splitting it into smaller bits in a way that avoids moves becoming apparent changes would really help.

**Comment by vinyldarkscratch** [comment]
I separated this PR into three other PRs, see #7647, #7648, and #7649.

**Pull request closed without merging**